### PR TITLE
Raise plugin- and/or role versions

### PR DESCRIPTION
**Pull request by tobias-richter**
Add/Update plugins for Jenkins 2.319.2

**Removed deprecated plugins:**

* pipeline-model-declarative-agent

### CODE REVIEW

File: defaults/main.yaml [hunk 1]
```diff
@@ -49,169 +49,194 @@ jenkins_pipeline_library_debug: false
...
     version: "1.1"
   - name: authentication-tokens
     version: "1.4"
+  - name: antisamy-markup-formatter
+    version: "2.7"
   - name: apache-httpcomponents-client-4-api
     version: "4.5.13-1.0"
   - name: bootstrap4-api
-    version: "4.6.0-3"
+    version: "4.6.0-3"#
```
Comment (bellackn) [inline defaults/main.yaml]: This "#" looks like a typo

File: defaults/main.yaml [hunk 2]
```diff
@@ -49,169 +49,194 @@ jenkins_pipeline_library_debug: false
 
 # Plugins and their versions that must be present for jenkins-pipeline-library
 jenkins_pipeline_library_plugins_present:
+  - name: analysis-model-api
```
Comment (bellackn) [inline defaults/main.yaml]: These are not alphabetically ordered. I know, "authentication-tokens" isn't either, but as we already touch it anyway...

File: meta/main.yml [hunk 3]
```diff
@@ -63,3 +63,4 @@ dependencies:
     jenkins_script_security_approved_signatures_present: "{{ jenkins_pipeline_library_approved_signatures_present }}"
     tags:
       - 'wcm_io_devops.jenkins_pipeline_library.jenkins_script_security'
+      - "wcm_io_devops.jenkins_script_security"
```
Comment (bellackn) [inline meta/main.yml]: This check for new line character at the end of file can be easily fixed

**Review by bellackn — CHANGES_REQUESTED** [review]

File: defaults/main.yaml (hunk 1 above)
Comment (tobias-richter) [reply defaults/main.yaml]: Solved

File: defaults/main.yaml (hunk 2 above)
Comment (tobias-richter) [reply defaults/main.yaml]: Solved

**Review by bellackn — APPROVED** [review]
lgtm

**Pull request merged**